mobile-app/app/screens/AppNavigator/screens/Portfolio/components/ActionButtons.tsx

```diff
@@ -27,7 +27,7 @@ export function ActionButtons (): JSX.Element {
   return (
     <View testID='action_button_group'>
       <ScrollView
-        contentContainerStyle={tailwind('px-5 mt-8 flex-row')}
+        contentContainerStyle={tailwind('flex flex-row justify-between w-full px-5 mt-8')}
```

Comment:
Can you help to test on mobile again? 
- on iOS, the scrolling to the right is not able to see the last button in full
- on android, the action button is not able to scroll horizontally

iOS: https://user-images.githubusercontent.com/40191153/182810805-3656e187-6209-4cf2-abae-f74c5c601863.MP4
android: https://user-images.githubusercontent.com/40191153/182811469-d0f8728b-52e7-43bb-9997-292fad8cb7e6.mp4